AdMobAdapter/src/main/java/com/chartboost/mediation/admobadapter/AdMobAdapter.kt

```diff
@@ -63,6 +66,24 @@ class AdMobAdapter : PartnerAdapter {
                 }
             }
 
+        /**
+         * Convert a given AdMob error code into a [ChartboostMediationError].
+         *
+         * @param error The AdMob error code as an [Int].
+         *
+         * @return The corresponding [ChartboostMediationError].
+         */
+        internal fun getChartboostMediationError(error: Int) =
```

Comment:
same as bidding. private okay here?

Reply:
no, the callback implementations are reference this, and they're in their own separate classes now.